src/util/fetch.js

```diff
@@ -32,6 +33,4 @@ export const fetchJson = (url, options = {}) => {
         });
 };
 
-export const queryParameters = data => Object.keys(data)
-    .map(key => [key, data[key]].map(encodeURIComponent).join('='))
-    .join('&');
+export const queryParameters = stringify;
```

Comment:
I suggest you also replace every call to `queryParameters` by a call to `stringify` in the code and documentation, and keep this `export` statement only for BC.

```
docs/Tutorial.md:587:    const { queryParameters } = fetchUtils;
docs/Tutorial.md:598:        url = `${API_URL}/${resource}?${queryParameters(query)}`;
docs/Tutorial.md:608:        url = `${API_URL}/${resource}?${queryParameters(query)}`;
docs/Tutorial.md:619:        url = `${API_URL}/${resource}?${queryParameters(query)}`;
src/rest/jsonServer.js:1:import { queryParameters, fetchJson } from '../util/fetch';
src/rest/jsonServer.js:45:            url = `${apiUrl}/${resource}?${queryParameters(query)}`;
src/rest/jsonServer.js:62:            url = `${apiUrl}/${resource}?${queryParameters(query)}`;
src/rest/simple.js:1:import { queryParameters, fetchJson } from '../util/fetch';
src/rest/simple.js:44:            url = `${apiUrl}/${resource}?${queryParameters(query)}`;
src/rest/simple.js:54:            url = `${apiUrl}/${resource}?${queryParameters(query)}`;
src/rest/simple.js:65:            url = `${apiUrl}/${resource}?${queryParameters(query)}`;
src/util/fetch.js:35:export const queryParameters = data => Object.keys(data)
```